staging/src/k8s.io/legacy-cloud-providers/aws/aws.go

```diff
@@ -3515,36 +3510,36 @@ func getSGListFromAnnotation(annotatedSG string) []string {
 // Extra groups can be specified via annotation, as can extra tags for any
 // new groups. The annotation "ServiceAnnotationLoadBalancerSecurityGroups" allows for
 // setting the security groups specified.
-func (c *Cloud) buildELBSecurityGroupList(serviceName types.NamespacedName, loadBalancerName string, annotations map[string]string) ([]string, error) {
+func (c *Cloud) buildELBSecurityGroupList(serviceName types.NamespacedName, loadBalancerName string, annotations map[string]string) ([]string, bool, error) {
```

Comment:
Might be more clear to return the default SG separately instead of the bool, that way you can check for its existence and its obvious that we're only passing that one to `setSecurityGroupIngress().`

Reply:
@nckturner ,

But here is the problem with that in case of SG is coming from c.cfg.Global.ElbSecurityGroup we would not be able to understand the difference in the calling function and based on the behavior we don't want to change the Ingress of Global SG.